calico_containers/calico_ctl/checksystem.py

```diff
@@ -66,7 +70,7 @@ def check_system(fix=False, quit_if_error=False):
 
     system_ok = (_check_kernel_modules(fix) and
                  _check_ip_forwarding(fix) and
-                 _check_docker_version())
+                 _check_docker_version(libnetwork))
```

Comment:
I thought about this a bit more after you were having trouble hitting the `_check_docker_version` function when one of the previous functions in this statement returned `False`.

I don't like it.  If `_check_kernel_modules` returns False, this variable will immediately be set to false and the other two functions will not be called.  If your system is not configured correctly, you should get full output for everything that is wrong.

Though it's not as pretty, I think the function return values should be stored as separate variables, then checked here.